cluster/addons/fluentd-elasticsearch/es-image/elasticsearch.yml

```diff
@@ -1,7 +1,385 @@
-path:
-  data: /data/data
-  logs: /data/log
-  plugins: /data/plugins
-  work: /data/work
-cluster:
-  name: kubernetes_logging
+##################### Elasticsearch Configuration Example #####################
```

Comment:
I'm assuming this was copied from somewhere.

1. Is it missing a copyright notice?
2. Since we're only actually using 5 lines from this file and the config is bundled into the image anyway, is there any reason not to just include the 5 lines we actually want plus a link to this example config file, wherever it currently lives? Users won't actually be able to customize this stuff given that it's packaged into the image, only future maintainers will.